src/_templates/react-native-typescript-screen/new/prompt.js

```diff
@@ -0,0 +1,14 @@
+// see types of prompts:
+// https://github.com/SBoudrias/Inquirer.js#prompt-types
+//
+// and for examples for prompts:
+// https://github.com/SBoudrias/Inquirer.js/tree/master/examples
+module.exports = [
+  {
+    type: "list",
+    name: "componentType",
+    message: "FC or Class?",
+    choices: ["FC", "Class"],
+    default: "FC"
+  }
+];
```

Comment:
I think we can remove this to try and enforce functional components

Reply:
Ryan said yes. I don't have enough RN exp to say no. 
![image](https://user-images.githubusercontent.com/3806031/68616358-21f22b80-0482-11ea-9eaf-1de3493d042b.png)